### ISSUE TITLE
Test against test262

### ISSUE THREAD

**Issue by jedel1043** [boa-dev/temporal#4]
Would be good to see if we can somehow use Boa to test this, but the cyclic dependency worries me.

**Comment by nekevss** [comment]
Agreed. It would also be good to have better unit testing across the board.

**Comment by nekevss** [comment]
Removed from 0.1.0 blocking. This would be nice in the long term, but temporal_rs is used in Boa, V8, and Keisel. So the general test coverage via proxy is pretty decent
